src/printer.js

```diff
@@ -1375,7 +1385,7 @@ function printPathNoParens(path, options, print) {
       if (isChainedElseIf) {
         return content
       }
-      return group(content, {shouldBreak})
+      return group(content, {shouldBreak, visibleType: 'if'})
```

Comment:
I'm surprised `if`s aren't `visibleType: 'visible'`?

If they should be, this could be a use case for allowing multiple visible types? Or could presumably achieve the equivalent by just nesting two groups with different visible types eg `group(group(..., {visibleType: 'if'}), {visibleType: 'visible'})`